components/mpas-ocean/cime_config/buildnml

```diff
@@ -103,8 +103,8 @@ def buildnml(case, caseroot, compname):
         decomp_date += '160830'
         decomp_prefix += 'mpas-o.graph.info.'
     elif ocn_grid == 'oEC60to30v3wLI':
-        ic_date += '170328'
-        ic_prefix += 'oEC60to30v3wLI'
+        ic_date += '171031'
+        ic_prefix += 'oEC60to30v3wLI60lev'
```

Comment:
should this be part of this PR?

Reply:
This has been part of [PR#2336](https://github.com/E3SM-Project/E3SM/pull/2336), but we wanted to incorporate changes specific to this grid configuration here so that it will run properly. We can remove this from this PR if preferred.

Reply:
No that's fine.  Just checking that it wasn't in here accidentally.